frontend/packages/pipelines-plugin/src/components/repository/RepositoryLinkList.tsx

```diff
@@ -50,10 +58,16 @@ const RepositoryLinkList: React.FC<RepositoryLinkListProps> = ({ pipelineRun })
         <>
           <dt>{t('pipelines-plugin~Commit id')}</dt>
           <dd>
-            <CopyToClipboard
-              value={plrLabels[RepositoryLabels[RepositoryFields.SHA]]}
-              data-test="pl-commit-id"
-            />
+            {shaURL ? (
+              <ExternalLink href={shaURL} data-test="pl-sha-url">
+                <GithubIcon title={shaURL} /> {shaURL}
+              </ExternalLink>
```

Comment:
`Url` shouldn't be the link text.
  
![image](https://user-images.githubusercontent.com/9964343/128981586-176899ee-0fea-4063-b4a3-d42c02f13911.png)

The link text should be the truncated commit id. (7 characters long as github shows it without trailing `...`)

Reply:
made truncated commit id